accelerator/apis.c

```diff
@@ -25,6 +25,20 @@ int apis_logger_release() {
   return 0;
 }
 
+status_t api_get_ta_info(char** json_result) {
+  status_t ret = SC_OK;
+
+  ret = ta_generate_info_serialize(json_result);
```

Comment:
Change the function name to `ta_get_info_serialize` would stick to the purpose of api.